Cut(eos_cli_config_gen): Removing 'null' as valid value of esp integrity and encryption from ip-security module (#4336)
aristanetworks/ansible-avd: ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

              route_map: static-to-bgp
    ```

> ### The old "null" option for esp integrity and encryption has been replaced with "disabled" from the ip_security module

Reviewer: ```suggestion
### `ip_security.sa_policies[].esp.integrity` and `.encryption` "null" option has been replaced with "disabled"
```

Author: updated

In AVD 4.0.0 we had "null" as a valid value for esp integrity and encryption to apply the null security profile and encryption under the ip-security configuration.

As of AVD 5.0.0 "null" option for esp integrity and encryption has been replaced with "disabled" to apply the null security profile and encryption under the ip-security configuration.

> === "Old data models"
> 
>     ```yaml
>     # Old data model
>     ip_security:
>       sa_policies:
>         name: Disabled
>         esp:
>           integrity: "null"
>           encryption: "null"
>     ```
> 
> === "New data model"
> 
>     ```yaml
>     # New data model
>     ip_security:
>       sa_policies:
>         name: Disabled
>         esp:
>           integrity: disabled
>           encryption: disabled
>     ```

Reviewer: (add an extra ``` at the end. I could not add it here)
```suggestion
```diff
 ip_security:
   sa_policies:
     name: Disabled
     esp:
-      integrity: "null"
-      encryption: "null"
+      integrity: disabled
+      encryption: disabled
```

Author: done
